src/ape/managers/__init__.py

```diff
@@ -9,38 +9,18 @@
 from .converters import ConversionManager
 from .networks import NetworkManager
 from .plugins import PluginManager
-from .project import DependencyManager, ProjectManager
+from .project import ProjectManager
 from .query import QueryManager
 
-# Wiring together the application
-_data_folder = _Path.home().joinpath(".ape")
-_project_folder = _Path.cwd()
-
 ManagerAccessMixin.plugin_manager = PluginManager()
-
-ManagerAccessMixin.dependency_manager = DependencyManager(data_folder=_data_folder)
-
 ManagerAccessMixin.config_manager = ConfigManager(
-    # Store all globally-cached files
-    DATA_FOLDER=_data_folder,
```

Comment:
It now defaults to `Path.home() / ".ape"` but can be changed; the tests still use a temp dir.

Reply:
Where is this specifically though? Couldn't find where it moved to

Reply:
In `ape/managers/config.py:ConfigManager.__init__.py`:

```python
def __init__(self, data_folder: Optional[Path] = None, request_header: Optional[dict] = None):
        if not data_folder and "APE_DATA_FOLDER" in os.environ:
            self.DATA_FOLDER = Path(os.environ["APE_DATA_FOLDER"])
        else:
            self.DATA_FOLDER = data_folder or Path.home() / ".ape"

        self.REQUEST_HEADER = request_header or {}
```

I made it also support this env var so that tests that use subprocess-runners can use the temporary data folder as well.